guides/lando-phpstorm.md

```diff
@@ -22,13 +22,35 @@ mailchimp:
 [PhpStorm](https://www.jetbrains.com/phpstorm/) is a popular code IDE for PHP
 and Drupal development. This video tutorial shows you how to set up PhpStorm with Xdebug.
 
-If you’ve got a local php installation (for example php 7.1 installed with homebrew on macOS) that listens on port 9000 you may need to change the containers php.ini port specification to another port (i.e. `xdebug.remote_port=9001`) and tell phpstorm to listen on that port. See also [Debugging Drupal 8 with PHPstorm and Lando on your Mac](https://www.austinprogressivecalendar.com/blog/debugging-drupal8-phpstorm-and-lando-your-mac).
+## Getting Started
 
-## Xdebug 3.x (PHP 7.3+)
-With Xdebug 3.x, the setting `xdebug.remote_port` has been deprecated, and the setting `xdebug.client_port` should be used instead.
-Also, the default xdebug port changed from `9000` to `9003`. Xdebug 3 is now the default version for PHP 7.3 and above.
+Enable Xdebug by adding the `xdebug: true` line to your `.lando.yml`.
+
+When using a recipe, add it under the `config` key:
+```yaml
+name: mywebsite
+recipe: drupal10
+config:
+  xdebug: true
```

Comment:
I've found it easiest to specify the `develop,debug` setting here, rather than using `true`.  Is there a reason to split it into two parts?

If so, that should be specified up front, since it is a Lando-supported feature.

Reply:
I think `develop,debug` is the actual result of using `true` here. I actually had it that way at first, but leaned toward simplicity since I think the goal is "get up and running" rather than getting into the features. The `true` seemed like it would be more obvious to a newcomer that we're just turning it on. The specific mode would require more explanation. I'm not sure what you mean by splitting it into two parts?

Reply:
It's non-obvious that `true` is an alias of `debug,develop`.  I didn't even realize that's what it meant. (But not `profile`?)  Which led me initially to lots of `xdebug: true` under config, and then `xdebug: debug,develop` in the appserver override block.  When I realized that was redundant, I rolled my eyes and switched to putting `xdebug: debug,develop` in the main config block and moving on with life.

I fear "true is simpler" may be true in the narrow sense, but also hide functionality from the user.  It's better to give them defaults that are "self-teaching" and scale up with them as needed, rather than one that may lead them down an incorrect path, as it did me.

Reply:
That's fair. In fact, support for "true" should probably be deprecated and dropped in v4 in favor of setting the mode every time.

Reply:
New commit added with this update and an added reference to Lando's Xdebug docs.